### ISSUE TITLE
DeltaChat desktop flatpak 1.46.0 suddenly fails to start

### ISSUE THREAD

**Issue by dln949** [flathub/chat.delta.desktop#136]
originally did not realize this is only a flatpak issue, originally posted here:  https://github.com/deltachat/deltachat-desktop/issues/3930

Delta Chat desktop flatpak 1.46.0, Ubuntu 22.04, Gnome shell 42.9, Wayland

Today I suddenly have the problem that deltachat-desktop refuses to start.

In the original github thread this was posted by Simon-Laux:  

I tried on ubuntu 22.04, I can reproduce this issue on a wayland session on an X session it works perfectly. I couldn't get it to work in weston, maybe flatpak and snap on ubuntu both have some problems with Weston and mistake it for a compositor based on Xorg (they complain about missing $DISPLAY env var and that seems to be an Xorg thing).

Current suggested workaround: use the snap version, that works. Or use the Debian (.deb package that is offered for download on the website) version, but I haven't tested that.

**Comment by ralphtheninja** [comment]
> Snap uses `--disable-gpu` flag: https://github.com/nbuechner/deltachat-snap/blob/44442f142c72fa0f26f2b85890ac6044b2a18abe/snapcraft.yaml#L27
> 
> Probably flatpak needs it too? GPU is not necessarily better, on my laptop I don't see the difference. This certainly should fix some problems for GPU users with bad drivers (e.g. nvidia).

Yes! I can confirm this seems to solve this issue https://github.com/deltachat/deltachat-desktop/issues/4676
